src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -935,7 +936,32 @@ public static String generateOriginalExtension(String fileType) {
         return null;
     }   // end createDataFiles
     
-    private static File saveInputStreamInTempFile(InputStream inputStream, Long fileSizeLimit) throws IOException, FileExceedsMaxSizeException {
+    public static boolean deleteDirectory(File dir) {
```

Comment:
why not just use `FileUtils.deleteDirectory(new File(destination));`?

Reply:
@oscardssmith - not sure what you're suggesting. FWIW: I just moved this method from ShapefileHandler since I added a use in FileUtils and it seemed like a general util function.

Reply:
Got it! I hadn't realized that ShapefileHandler already had that import and was using the apache class. I've deleted the method here and switched to apache.